### PR TITLE
Fetch network element and filter by nominal voltages

### PR DESCRIPTION
**Pull request by souissimai**

### CODE REVIEW

File: src/services/study/network.js [hunk 1]
```diff
@@ -138,16 +139,21 @@ export function fetchNetworkElementsInfos(
     substationsIds,
     elementType,
     infoType,
-    inUpstreamBuiltParentNode
+    inUpstreamBuiltParentNode,
+    nominalVoltages = null
```
Comment (sBouzols) [inline src/services/study/network.js:143]: undefined ?

**Review by YenguiSeddik — APPROVED** [review]

File: src/services/study/network.js (hunk 1 above)
Comment (sBouzols) [reply src/services/study/network.js:143]: see `fetchEquipmentsIds` in network-map.js

Comment (sBouzols) [reply src/services/study/network.js:143]: see #2141

Comment (AAJELLAL) [reply src/services/study/network.js]: Done

File: src/services/study/network.js [hunk 2]
```diff
@@ -138,16 +139,21 @@ export function fetchNetworkElementsInfos(
     substationsIds,
     elementType,
     infoType,
-    inUpstreamBuiltParentNode
+    inUpstreamBuiltParentNode,
+    nominalVoltages = undefined
 ) {
     console.info(
-        `Fetching network '${elementType}' elements '${infoType}' infos of study '${studyUuid}' and node '${currentNodeUuid}' with substations ids '${substationsIds}'...`
+        `Fetching network '${elementType}' elements '${infoType}' infos of study '${studyUuid}' and node '${currentNodeUuid}' with ${substationsIds?.length} substations ids.`
```
Comment (sBouzols) [inline src/services/study/network.js:146]:
```
    const substationsCount = substationsIds ? substationsIds.length : 0;
    const nominalVoltagesStr = nominalVoltages ? `[${nominalVoltages}]` : '[]';
 
    console.info(
        `Fetching network '${elementType}' elements '${infoType}' infos of study '${studyUuid}' and node '${currentNodeUuid}' for ${substationsCount} substations ids and ${nominalVoltagesStr} nominal voltages.`
    );
 ```

Comment (AAJELLAL) [reply src/services/study/network.js]: Done

**Review by sBouzols — APPROVED** [review]
Code Review OK
Test OK 
alea creation from map for lines with multiple conf for nominalVoltages filter
tested with no nominalvoltage selection, all, some combination
Console Warning check OK

**Review by AAJELLAL — APPROVED** [review]
Code and tests are OK.

**Pull request merged**